cloudinit/util.py

```diff
@@ -2316,10 +2316,6 @@ def make_header(comment_char="#", base="created"):
     return header
 
 
-def abs_join(base, *paths):
```

Comment:
Why are we duplicating this multiple times now?

Reply:
Right, I didn't bother to explain that tradeoff.

My argument here is: "packaging code, and especially packaging code that is only used for building test packages, should not be shipped as part of the cloud-init runtime".

If we agree on that point we have a few options:

1. duplicate (chosen)
2. replace this one function call with the two function calls it simplifies at callsites 
3. create a new module to store shared code

Option 2 involved more typing. Option 3 seemed like overkill for a one line function. Option 1 seemed like a reasonable tradeoff, but I will happily do 2 if preferred. Option 3 sounds like a refactor to make it worthwhile. If there are other sharable/shared functions to consolidate I guess we can do that too.

Reply:
@TheRealFalcon alright, per out of band convo it sounds like 2 is preferred.